src/components/centered/centered.scss

```diff
@@ -0,0 +1,31 @@
+.page {
+  align-items: center;
+  background-color: $wolox-blue;
+  background-image: url('./../assets/wolox_bg.png');
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 100px 0;
+}
+
+.logo {
+  max-width: 350px;
+  margin-bottom: 50px;
+}
+
+.title {
+  color: $white;
+  font-size: 30px;
+  font-weight: 500;
+  font-family: Montserrat, sans-serif;
```

Comment:
This font family should be attached to the body

Reply:
I want only the title to have that font. The rest has sans-serif.